src/nxdoc/getting-started/tracks/first-package-with-studio.md

```diff
@@ -454,3 +454,23 @@ CUSTOMIZE THE LOGIN PAGE
 {{#> callout type='tip' }}
 Our teams are working round the clock to improve the themes and styling of Web UI so that you can apply your own branding. Check in again soon to see the improvements we've made and to further customize your Nuxeo application.
 {{/callout}}
+
+## Application Definition
+
+INSTALL NEW PACKAGES
+
+1. In Studio, select **Application Definition** from the **Settings** menu.
+   ![]({{file name='AppDef1.png' version= '1010' page='first-package-with-studio'}} ?w=450,border=true)
+
+2. Enter the package name you are looking for in the **Package to Install** input.
+   ![]({{file name='AppDef2.png' page='first-package-with-studio'}} ?w=450,border=true)
+   
+3. Select it. You should have a summary about packages changes.
+   ![]({{file name='AppDef3.png' page='first-package-with-studio'}} ?w=450,border=true)
+
+4. Save your changes, you should see that your package is installed.
+   ![]({{file name='AppDef4.png' page='first-package-with-studio'}} ?w=450,border=true)
```

Comment:
Technically, it's better to say that "The package will be installed when installing the Studio package and its registries are available in the Studio project", this last part is the important part to show for me